engine/runtime/src/test/scala/org/enso/compiler/test/semantic/ImportExportTest.scala

```diff
@@ -450,12 +591,17 @@ class ImportExportTest
         .createModule(packageQualifiedName.createChild("Main"))
         .getIr
 
+      mainIr.imports.size shouldEqual 1
+      mainIr.imports.head.isInstanceOf[errors.ImportExport] shouldBe true
       mainIr.imports.head
         .asInstanceOf[errors.ImportExport]
         .reason
-        .message should include(
-        "The symbol Impl (module or type) does not exist in module Test.Logical_Export.Main."
-      )
```

Comment:
Why are we no longer testing what the message is?

I'd like the tests to verify that the message is informative and comprehensible.

Reply:
The message is defined in https://github.com/enso-org/enso/blob/dec8dac8a01d29f9a3a18431ddc5179142a97167/engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/expression/errors/ImportExport.scala#L158-L164
 and is likely to change from time to time - which is what happened in this PR. This `ImportExportTest.scala` tests that the errors are of `SymbolDoesNotExist` type and checks the `symbolName` field. Repeating the message constant ten times in this test does not seem as a good idea to me.

Reply:
Ok